Team9Project/src/com/indragie/cmput301as1/ExpenseItemAddActivity.java

```diff
@@ -174,6 +186,30 @@ public void onClick(View v) {
 		});
 		
 		incomplete = false;
+		
```

Comment:
You can just select the code and do Ctrl+Shift+F in Eclipse to indent all of it correctly.